src/content/blog/2024-08-27-integration-tests.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Leveling Up Rafiki Testing: Shifting from Manual to Automated"
+description: "How we automated our manual payment flow tests."
+date: 2024-08-27
+slug: integration-tests
+author: Blair Currey
+author_url: https://www.linkedin.com/in/blair-currey/
+tags:
+  - Interledger
+  - Rafiki
+  - Open Payments
+  - Testing
```

Comment:
How are these used? wasn't really sure what would be most appropriate here

Reply:
They are not used yet but they will be in the future when we have enough content. 
I'd delete Interledger from your list here.